packages/data-models/src/driver.js

```diff
@@ -36,6 +36,9 @@ import WordList from './word-list.js'
 import TreebankDataItem from './treebank_data_item.js'
 import Logger from './logging/logger.js'
 
+import L10n from './l10n/l10n.js'
```

Comment:
From my point of view it would be better to create a separate l10n package,
and the locale file (with translations) should stay inside the package they belong to.
I see the following reasons for that:
   - this way it could be used inside any alpheios application (even if it doesn't need the whole alpheios-core)
      for example may be it is useful to use it in a new alignment editor library, but I want to avoid to upload there the whole huge alpheios-core
    - we could store localization files withing the packages to simplify there handling and navigation
    - l10n has no dependencies for other packages (including data-modules) so it could became really a small and light package

Reply:
I agree that it's better to have `L10n` as a separate package. @balmas, what do you think?

Reply:
yes that seems reasonable.